sdks/python/container/py39/base_image_requirements.txt

```diff
@@ -82,11 +82,13 @@ hypothesis==6.92.1
 idna==3.6
 importlib-metadata==7.0.0
 iniconfig==2.0.0
+Jinja2==3.1.2
 joblib==1.3.2
 Js2Py==0.74
 jsonpickle==3.0.2
 jsonschema==4.20.0
 jsonschema-specifications==2023.11.2
+MarkupSafe==2.1.3
```

Comment:
MarkupSafe version 2.1.0 has a breaking change that broke a bunch of libs including Jinja2, so pinning a higher version here